### PR TITLE
Update for 8.0 GA (stable versions)

### PR DESCRIPTION
**Pull request by DamianEdwards**
Updating samples for 8.0 GA release using stable versions.

This will need the nuget.config and CI changes reverted before merging.

### CODE REVIEW

**Review by eerhardt — APPROVED** [review]
Seems good. Just version updates (sans the changes that should be reverted).

**Review by joperezr — APPROVED** [review]
LGTM. Thanks @DamianEdwards

File: NuGet.config [hunk 1]
```diff
@@ -2,9 +2,21 @@
...
   </packageSources>
   <solution>
     <add key="disableSourceControlIntegration" value="true" />
   </solution>
+  <packageSourceMapping>
+    <clear />
+    <packageSource key="aspire-stable">
+      <package pattern="Aspire.*" />
+      <package pattern="Microsoft.Extensions.ServiceDiscovery.*" />
+      <package pattern="Microsoft.Extensions.ServiceDiscovery" />
```
[the remark this answers is not in the document]
Comment (joperezr) [reply NuGet.config:16]: It should, yeah.

[the remark this answers is not in the document]
Comment (DamianEdwards) [reply NuGet.config:16]: I thought I tried it and it didn't, but this will get reverted anyway :smile:

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -32,6 +32,7 @@ jobs:
       DOTNET_CLI_TELEMETRY_OPTOUT: true
       DOTNET_NOLOGO: true
       DOTNET_MULTILEVEL_LOOKUP: false
+      DOTNET_CLI_WORKLOAD_UPDATE_NOTIFY_DISABLE: true
```
Comment (joperezr) [inline .github/workflows/ci.yml:35]:
```suggestion
```

**Pull request merged**